src/pykiso/lib/connectors/cc_rtt_segger.py

```diff
@@ -108,6 +109,7 @@ def __init__(
         :param rtt_log_speed: number of log per second to be pulled (manage the CPU load for logging)
             None value fetch log at the CPU's speed. Default 1000 logs/s
         :param connection_timeout: available time (in seconds) to open the connection
+        :param search_range: range to search for the block of the device
```

Comment:
Please add docs

For me its not clear if search_range is just an address or is it a length in bytest beginning at  block_address